graveler/ref/repository_iterator_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/treeverse/lakefs/testutil"
 )
 
-func TestPGRepositoryIterator(t *testing.T) {
+func TestRepositoryIterator(t *testing.T) {
 	r, db := testRefManagerWithDB(t)
 	repos := []graveler.RepositoryID{"a", "aa", "b", "c", "e", "d"}
```

Comment:
6 is still divisible by 3, I'd rather we also tested a partial fetch.

Reply:
fixed